extensions/notification/NotificationConfigValidator.js

```diff
@@ -1,27 +1,45 @@
 const joi = require('joi');
 const { probeServerJoi } = require('../../lib/config/configItems.joi');
 
-const authSchema = joi.object({
-    type: joi.string(),
-    ssl: joi.boolean(),
-    protocol: joi.string(),
+const sslSchema = joi.object({
+    ssl: joi.boolean().default(false),
```

Comment:
Would having `type: "ssl"` be more consistent with the other auth types?

Reply:
The ssl auth isn't really a distinct type, it can be used with kerberos and basic auth as well. My feeling is most deploys using kerberos and basic auth will use at least `ca` and `key` so the credentials aren't sent over the wire unencrypted.